changelog

```diff
@@ -1,14 +1,110 @@
+Version 0.9.11:
+  * Big changes:
+    * Added the Graveyard, a region near the Ember Waste, and its inhabitants. (@beccabunny)
+  * Bug fixes:
+    * Typo fixes. (@alextd, @Amazinite, @Anarchist2, @Arachi-lover, @Darcman99, @Gods-Righthand, @Mageking17, @MCOfficer, @oo13, @otsohelos, @perey, @petervdmeer, @tehhowch, @Waffleship, @wjp, @Zitchas)
```

Comment:
Forgot @Jozephi-Krakowskije (#4650)

Reply:
The PR that fixed that issue was from you though, and as far as I’m aware the change log has always mentioned PR authors, not issue authors.

Reply:
then i'm surprised to see otsohelos here, unless they also contributed another PR.

Reply:
I went to the released page and where it says “X commits since last release,” I clicked that and scrolled through all the commits. So if someone is mentioned here, it means that authored a commit.